main/gdb/APKBUILD

```diff
@@ -12,7 +13,6 @@ makedepends="ncurses-dev expat-dev texinfo readline-dev python3-dev
 options="!check"
 subpackages="$pkgname-doc"
 source="https://ftp.gnu.org/gnu/$pkgname/$pkgname-$pkgver.tar.xz
-	s390x-use-elf-gdb_fpregset_t.patch
```

Comment:
I think this patch needs to be rebased instead of removed. It fails to build on s390x:

````
proc-service.o: In function `ps_lgetfpregs':
(.text+0x312): undefined reference to `fill_fpregset(regcache const*, elf_fpregset_t*, int)'
proc-service.o: In function `ps_lsetfpregs':
(.text+0x3b0): undefined reference to `supply_fpregset(regcache*, elf_fpregset_t const*)'
collect2: error: ld returned 1 exit status
````

@tmh1999 can you please have a look at this?

Reply:
fixed up